content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,180 @@
+should be noted that :abbr:`MRP (Material Requirement Planning)` devices are not compatible. *These
+include cameras or measurement tools.*
+
+Pre-requisites
+==============
+
+The following items will be needed to complete the Windows IoT installation.
+
+- Odoo 16 database or any version above.
+- :abbr:`IoT (Internet of Things)` compatible devices (except those mentioned above). Refer to:
+  `Odoo's compatible IoT devices <https://www.odoo.com/app/iot-hardware>`_.
+- Device drivers for Windows.
+
+  .. note::
+     Odoo recommends using an updated, recent version of Windows as some older operating systems can
+     cause the Windows virtual :abbr:`IoT (Internet of Things)` to not work.
+
+- Windows computer (laptop, desktop, server).
+- Odoo :abbr:`IoT (Internet of Things)` subscription. Refer to: :ref:`iot/iot-eligibility`.
+
+Connect the Windows virtual Iot box to an Odoo database
+=======================================================
+
+The Windows virtual IoT box is simple to setup in just a few easy steps. Follow this process when
+installing the Windows virtual IoT software for the first time.
+
+Download and initial installation
+---------------------------------
+
+To begin the installation, navigate to the Odoo 16 or higher installation package for Enterprise or
+Community Windows at `Odoo's Download Page <https://odoo.com/download>`_. Next, Install and setup
+the Odoo `.exe` file. After the instructions screen, click :guilabel:`Next` to start the
+installation and agree to the :abbr:`TOS (Terms of Service)`.
+
+During the next step of the installation, select :guilabel:`Odoo IoT` from the :guilabel:`Select the
+type of install` dropdown.
+
+.. example::
+   For reference, the following should be installed:
+
+   - **Odoo server**
+   - **Odoo IoT**
+   - **Nginx WebServer**
+   - **Ghostscript interpreter**
+
+Ensure there is enough space on the computer for the installation and click :guilabel:`Next`.
+
+Setting the destination and completing the installation
+-------------------------------------------------------
+
+To complete the installation, select the :guilabel:`Destination Folder`. Choosing ``C:\odoo`` as the
+install location will allow for the Nginx server to start. Click :guilabel:`Install`. If the folder
+doesn't exist, then create it. Otherwise the installation files will be spread throughout the hard
+drive. Allow the installation to occur. This may take a few minutes. Click :guilabel:`Next` to
+continue.
+
+.. warning::
+   Odoo's Windows virtual IoT software shouldn't be installed inside any of the Window's User's
+   directories. Doing so won't allow for Nginx to initialize.
+
+Ensure that the :guilabel:`Start Odoo` box is checked and click :guilabel:`Finish`. After
+installation, the Odoo server will run and automatically open `http://localhost:8069` on your web
+browser. The webpage should display the :abbr:`IoT (Internet of Things)` *box home page*
```

Comment:
Missing period and can remove italic formatting
```suggestion
browser. The webpage should display the :abbr:`IoT (Internet of Things)` box home page.
```

Reply:
@StraubCreative This is the label of the page, I originally didn't have this italicized, but the expert pointed this out. I'm okay with either way, not a big deal in my opinion. We will have to implement in the IoT bundle PR if there are changes though (PR  #3673). I can double check the label as well on this.